packages/swc/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@rollup/plugin-swc",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Transpile JavaScript/TypeScript code with swc.",
+  "license": "MIT",
+  "repository": {
+    "url": "rollup/plugins",
+    "directory": "packages/swc"
+  },
+  "author": "Peter Placzek <peter.placzek1996@gmail.com>",
+  "homepage": "https://github.com/rollup/plugins/tree/master/packages/swc#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/cjs/index.js",
+  "module": "dist/es/index.js",
+  "exports": {
+    "types": "./types/index.d.ts",
+    "import": "./dist/es/index.js",
+    "default": "./dist/cjs/index.js"
+  },
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "ci:coverage": "nyc pnpm test && nyc report --reporter=text-lcov > coverage.lcov",
+    "ci:lint": "pnpm build && pnpm lint",
+    "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
+    "ci:test": "pnpm test -- --verbose && pnpm test:ts",
+    "prebuild": "del-cli dist",
+    "prepare": "if [ ! -d 'dist' ]; then pnpm build; fi",
+    "prerelease": "pnpm build",
+    "pretest": "pnpm build",
+    "release": "pnpm --workspace-root plugin:release --pkg $npm_package_name",
+    "test": "ava",
+    "test:ts": "tsc types/index.d.ts test/types.ts --noEmit"
+  },
+  "files": [
+    "dist",
+    "!dist/**/*.map",
+    "src",
+    "types",
+    "README.md"
+  ],
+  "keywords": [
+    "rollup",
+    "plugin",
+    "swc",
+    "speedy-web-compiler",
+    "npm",
+    "modules"
+  ],
+  "peerDependencies": {
+    "@swc/core": "^1.x",
+    "rollup": "^2.x || ^3.x"
+  },
+  "peerDependenciesMeta": {
+    "rollup": {
+      "optional": true
+    }
+  },
+  "dependencies": {
+    "smob": "^0.0.6"
```

Comment:
I searched the repo and noticed `node-resolve` is using `deepmerge` for the same functionality. Is there a reason yo chose `smob` specifically? Otherwise maybe we should use `deepmerge` for consistency?

Reply:
`deepmerge` does not provide type definition by default, which is a disadvantage, because you have to install `@types/deepmerge` additionaly. Besides, deepmerge only accepts **one** source, while smob accepts **multiple** sources to be merged to a target object. Furthermore, with smob a custom merger can be created, which allows to customize the merge behavior in certain situations.

Reply:
Smob is also a zero dependency library and is used in the terser plugin as well ;)